src/main/java/nl/knaw/dans/avbag/EasyPreprocessAvBag.java

```diff
@@ -44,15 +44,12 @@ public String getName() {
 
     @Override
     public void configureCommandLine(CommandLine commandLine, EasyPreprocessAvBagConfig config) {
-        try {
-            commandLine.addSubcommand(new ConvertCommand(
-                new PseudoFileSources(config.getPseudoFileSources()),
-                config.getStagingDir()
-            ));
-        }
-        catch (IOException e) {
-            throw new RuntimeException(e);
-        }
+        log.info(System.getProperty(EXAMPLE_CONFIG_FILE_KEY));
+        log.info(System.getProperty(CONFIG_FILE_KEY));
```

Comment:
```suggestion
        log.debug(System.getProperty(EXAMPLE_CONFIG_FILE_KEY));
        log.debug(System.getProperty(CONFIG_FILE_KEY));
```

Reserve info logging for messages you expect that the end-user wants to see by default.